scripts/update-translations.sh

```diff
@@ -37,7 +37,7 @@ echo "Gathering all translation strings.  Note that this will probably take a wh
 ./manage.py makemessages --all --ignore 'corehq/apps/app_manager/tests/data/v2_diffs*' --ignore 'node_modules'
 
 echo "Gathering javascript translation strings.  This will also probably take a while"
-./manage.py makemessages -d djangojs --all
+./manage.py makemessages -d djangojs --all --ignore 'corehq/apps/app_manager/tests/data/v2_diffs*' --ignore 'node_modules' --ignore 'bower_components'
```

Comment:
Right now there are a bunch of `node_modules` strings in the `.po` file, but these will go away after I run `./scripts/update_translations.sh` (after this is merged).
Is there anything else we should be skipping over?  See anything in https://github.com/dimagi/commcare-hq/blob/master/locale/en/LC_MESSAGES/djangojs.po that doesn't belong?